department-of-veterans-affairs/abd-vro
# Mcp 1875 - Fix build warning/error

Fix some build warnings

## Description

Current build process produces some warning/error messages, though they don't fail the build. This PR fixes some issues
related to the warning/error messages as described in Jira ticket,  MCP-1875. They are:

1. Execution optimizations have been disabled for task ':app:dockerPrepare' to ensure correctness due to the following reasons: .....
2.  Configure project :app
Build was configured to prefer settings repositories over project repositories but repository 'MavenRepo' was added by build file 'app/build.gradle'
3. Task :app:checkstyleTest   abd-vro/app/src/test/java/gov/va/vro/VroControllerTest.java:4:47: Using the '.*' form of import should be avoided - org.junit.jupiter.api.Assertions.*. [AvoidStarImport]

Note: When 2. is addressed, similar warning for the code in BuildSrc shows up. This PR will not address any issue in BuildSrc and other codes that are not really used in building the images for this application.

### What was the problem?

Warning messages are shown in the build process. See above 1, 2, 3.

### How does this fix it?

Adjust codes to address the warnings.

### Jira Tickets

MCP-1875

## How to test this PR

- Step 1. build the application.
- Step 2. Check the build messages. The warning messages (1, 2, 3 above) discussed in MCP-1875 should not be seen.

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
